### PR TITLE
New clothing options for Chaplains [FIXED].

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
Added some new (mostly old actually) clothes for Chaplains. Accessible through PietyVend. Stuff added:
- New hats: kippah, wide-brimmed hat, cardinal's hat, bishop's mitre, witch hunter hat, pilgrim hat.
- New neck pieces: chaplain's stole (purple scarf thingy), crucifix.
- New coats: cardinal's robes (has toggleable hood), bishop's robes, witch hunter's coat (has toggleable hood).
- New jumpsuits: pilgrim's jumpsuit.
- New belt: chaplain's sash, that can hold job related items (censer, bible, bottle, crucifix, wooden stake).
- Misc items: censer (can be filled with ground tobacco/cannabis and lit for exactly zero benefits), wooden stake (for, uh, camping in the woods, I swear. Need to emag PietyVent to get it though. Or craft it.), mortuary urn (can store ash/ectoplasm, can attach paper label).

## Why / Balance
RP. Smoking censer! RP!

## Technical details
New .yml and .rsi, changes to PietyVend catalogue.

## Media
image
Fig. 1 - New clothes.
image
image
Fig. 2 - Mortuary urn.
- [X] I have added screenshots/videos to this PR showcasing its changes ingame.

**Changelog**
:cl: erhardsteinhauer
- add: Added new clothes for Chaplains. Check nearest PietyVend.

### CODE REVIEW

**Comment by ErhardSteinhauer** [comment]
Fixed YAML errors.

**Comment by ErhardSteinhauer** [comment]
And made wooden stake craftable.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/wooden_stake.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - type: Sprite
+    sprite: _NF/Objects/Weapons/Melee/wooden_stake.rsi
+    size: 2
+    state: icon
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    attackRate: 1.5
+    damage:
+      types:
+        Piercing: 8
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/wooden_stake.yml:20]: How is this compering to other stuff consider you can build it for cheap
This vs glass shiv for say?

Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/wooden_stake.yml:20]: OP, needs nerfing.

Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/wooden_stake.yml:20]: I think glass shiv is 5.5 of slashing damage. And brand new crowbar from vendomat is 10 brute dmg.
But I'd also argue that wooden plank is less readily available than glass.

Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Melee/wooden_stake.yml:20]: Eh, I'll bring it down to 6. This thing should not be equal to Crusher Dagger.

**Comment by Everturning** [comment]
are the stakes able to be thrown? possibly make it so that it's a throwable weapon that disappears on contact (stake gets lodged deep in the enemy, farther than can be pulled out), but up its damage somewhat

**Comment by ErhardSteinhauer** [comment]
> are the stakes able to be thrown? possibly make it so that it's a throwable weapon that disappears on contact (stake gets lodged deep in the enemy, farther than can be pulled out), but up its damage somewhat

Stakes are throwable and they break when you try to pull them out.

**Review by dvir001 — APPROVED** [review]
All working fine now

**Pull request merged**